_layouts/home.html

```diff
@@ -13,12 +13,99 @@ <h2 id="countdown">&nbsp;</h2>
+                  <li data-target="#bootstrap-touch-slider" data-slide-to="0" class="active"></li>
+                  <li data-target="#bootstrap-touch-slider" data-slide-to="1"></li>
+                  <li data-target="#bootstrap-touch-slider" data-slide-to="2"></li>
+              </ol>
+
+              <!-- Wrapper For Slides -->
+              <div class="carousel-inner" role="listbox">
+
+                  <!-- Third Slide -->
+                  <div class="item active">
+
+                      <!-- Slide Background -->
+                      <img src="../img/homepage_slide#1.jpg" alt="Bootstrap Touch Slider"  class="slide-image"/>
+                      <div class="bs-slider-overlay"></div>
+
+                      <div class="container">
+                          <div class="row">
+                              <!-- Slide Text Layer -->
+                              <div class="slide-text slide_style_left">
+                                <h1 data-animation="animated zoomInRight">Looking forward to meet you in our conference!!!</h1>
+                                <p data-animation="animated fadeInLeft">The best is yet to come!!!</p>
+                                <a href="http://bootstrapthemes.co/" target="_blank" class="btn btn-default" data-animation="animated fadeInUp">Become an host</a>
+                                <a href="http://bootstrapthemes.co/" target="_blank"  class="btn btn-primary" data-animation="animated fadeInDown">Attend an event</a>                              </div>
+                          </div>
+                      </div>
+                  </div>
+                  <!-- End of Slide -->
+
+                  <!-- Second Slide -->
+                  <div class="item">
+
+                      <!-- Slide Background -->
+                      <img src="../img/homepage_slide#2.jpg" alt="Bootstrap Touch Slider"  class="slide-image"/>
+                      <div class="bs-slider-overlay"></div>
+                      <!-- Slide Text Layer -->
+                      <div class="slide-text slide_style_center">
+                        <h1 data-animation="animated zoomInRight">Looking forward to meet you in our conference!!!</h1>
+                        <p data-animation="animated fadeInLeft">The best is yet to come!!!</p>
+                        <a href="http://bootstrapthemes.co/" target="_blank" class="btn btn-default" data-animation="animated fadeInUp">Become an host</a>
+                        <a href="http://bootstrapthemes.co/" target="_blank"  class="btn btn-primary" data-animation="animated fadeInDown">Attend an event</a>
+                      </div>
+                  </div>
+                  <!-- End of Slide -->
+
+                  <!-- Third Slide -->
+                  <div class="item">
+
+                      <!-- Slide Background -->
+                      <img src="../img/homepage_slide#3.jpg" alt="Bootstrap Touch Slider"  class="slide-image"/>
```

Comment:
where do these images come from? Are they CC licensed? Or Public Domain? Even if this is an early prototype for now, we can't have copyrighted material in our source.